openy_calc/openy_calc.info.yml

```diff
@@ -9,3 +9,4 @@ dependencies:
   - openy_socrates:openy_socrates
   - openy_mappings:openy_mappings
   - openy_node_mbrshp:openy_node_mbrshp
+  - lb_branch_hours_blocks:lb_branch_hours_blocks
```

Comment:
We need to add hook_update_N with module enabling every time we add dependency to info file

Reply:
Added hook_update_N